src/analog/commands/uplink/WriteImageResponse.ts

```diff
@@ -0,0 +1,94 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
+import {UPLINK} from '../../constants/directions.js';
+
+
+/**
+ * WriteImageResponse command parameters
+ */
+interface IWriteImageParameters {
+    offset: number,
+    status: boolean
+}
+
+
+const COMMAND_ID = 0x2a1f;
+const COMMAND_SIZE = 5;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'write image',
+        parameters: {
+            offset: 4,
+            status: true
```

Comment:
`status` already declared and used as number, like [here](https://github.com/jooby-dev/jooby-codec/blob/main/src/analog/commands/uplink/SetTime2000Response.ts#L13) 

for sameness, it is better to leave the number, or discuss and change it to a Boolean in other commands too

Reply:
fixed